File: rb/test/exists_test.rb
# Api2newsEndpoint SDK exists test

require "minitest/autorun"
require_relative "../Api2newsEndpoint_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = Api2newsEndpointSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- Api2newsEndpoint SDK exists test

local sdk = require("api2news-endpoint_sdk")

describe("Api2newsEndpointSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
